src/opnsense/www/js/widgets/OpenVPNClients.js

```diff
@@ -116,7 +116,7 @@ export default class OpenVPNClients extends BaseTableWidget {
                                    title="${client.status}">
                                 </i>
                                 &nbsp;
-                                <a href="/ui/openvpn/status">${client.common_name}</a>
+                                <a href="/ui/openvpn/status#search=${client.common_name}">${client.common_name}</a>
```

Comment:
The client connections are grouped by server, which has a "no clients connected" clickable link case as well. One could argue this server may also be filtered on.

https://github.com/opnsense/core/pull/8115/files#diff-d318e5384c1c7d42653b13d0fb1e444dd7a2c7251e587989a52364082cac953eR147

Reply:
Added :)